### PR TITLE
Hello-world project

### PR DESCRIPTION
**Pull request by Isabellanamutenda**
I have made all the changes that I was asked to make and 
I also solve the linter error.
I deleted the gitignore.md and replaced with .gitignore
I also made changes on the README file.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,33 @@
-# Hello-microverse
+# Hello World
+
+
+This Hello world project is the first project where I  set up HTML & CSS linters
+
+
+![screenshot](./app_screenshot.png)
```
Comment (abruzy) [inline README.md:7]: Good job 👍🏽 

kindly fix the screenshot to show in the preview

File: README.md [hunk 2]
```diff
@@ -1 +1,33 @@
...
+
+
+## Getting Started
+
+      -Create a github repository
+      -create the folder
+      -create a gitignore file and a README file
+      -add it and commit it
+      -set up the linters of html and css
```
Comment (abruzy) [inline README.md:16]: Good job 👍🏽 
this is not proper way to setup the app on the person machine. You want to follow this step instead

- [ ] clone the repo
- [ ] install dependencies

And so on

kindly fix that

**Review by abruzy — CHANGES_REQUESTED** [review]
Hi @Isabellanamutenda   👏 , 

### Awesome job so far!! :+1: 

#### You have done wonderfully well, you passed `almost` all the checklist for this project :+1: just a few issues left to work on............ :slightly_smiling_face: 

# Status: Required Changes! ♻️ 
_1. Kindly check the comments under the review._
_2. Let's make your repo looks professional by applying one of the  5 professional repo basic rules, kindly update your `README` file.
Checkout out the below link for help, feel free to follow this [template](https://github.com/microverseinc/readme-template) from pathwright_
_3. delete the gitignore.md file you created, you create ```.gitignore``` and add test.md and node_modules inside the file.
_4. fix the linting errors as well

### ♻️ Kindly re-submit for review after making all those changes :slightly_smiling_face: Thumps -up await you.....

### Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (Isabellanamutenda) [reply README.md]: Resolved

File: README.md (hunk 2 above)
Comment (Isabellanamutenda) [reply README.md:15]: Resolved

File: gitignore.md [hunk 3]
```diff
@@ -0,0 +1 @@
+test.md should be ignored by git
```
Comment (voscarmv) [inline gitignore.md:1]: - [ ] This file should be names `.gitignore`, *not* `gitignore.md`, please fix this.
- [ ] The content of this file should be:
```
test.md
node_modules
```

File: index.html [hunk 4]
```diff
@@ -0,0 +1,12 @@
...
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Hello Microverse</title>
+    <link href="style.css" rel="stylesheet">
+</head>
+<body>
+   <header><div class =" title">Hello Microverse</div></header> 
+</body>
+</html>
```
Comment (voscarmv) [inline index.html:12]: - [ ] **[OPTIONAL]** You may use 2-space indentation in your HTML files in order to keep with best HTML practices.

**Review by voscarmv — CHANGES_REQUESTED** [review]
Hi @Isabellanamutenda , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

 ## Required Changes ♻️

- [ ] Please add a proper Pull Request description. This description may contain a summary of the concepts you learned in completing this project. Currently, your PR description is empty, please fix this! 
 image
- [ ] Your linters are still throwing errors in this PR. Please fix your `style.css` file:
image
- [ ] Your `.gitignore` file requires some changes which were specified previously but were not carried out correctly.

_Check the comments under the review._

## Optional suggestions

- We may try to start implementing best HTML practice early. I've left one optional suggestion for this.

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css [hunk 5]
```diff
@@ -0,0 +1,3 @@
+.title{
+    color: aqua;
+    }
```
Comment (voscarmv) [inline style.css:3]: - [ ] Your `style.css` file is showing the following linter mistakes. Please fix them:

image

**Review by voscarmv — COMMENTED** [review]
@Isabellanamutenda  here are some additional code comments I missed on my previous review. Cheers!
- [ ] These are the 4 linter errors still showing up in the Pull Request. Please fix them before submitting again!

File: gitignore.md (hunk 3 above)
Comment (Isabellanamutenda) [reply gitignore.md:1]: I have a question  on the instructions we were told to name it .gitignore.md should I really change it.

Comment (Isabellanamutenda) [reply gitignore.md]: Resoolved

File: style.css (hunk 5 above)
Comment (Isabellanamutenda) [reply style.css]: Resolved

**Review by Emmyn5600 — APPROVED** [review]
## STATUS: APPROVED :muscle::clap:

### Hi, @Isabellanamutenda 

Congratulations! :tada:
Your project is complete!  It's time to merge it :shipit:
Great work on implementing the changes requested by the previous code reviewers :clap:

### Highlights
   * #### You have the descriptive Readme:heavy_check_mark:
   * #### You have descriptive commits :heavy_check_mark:
   * ####  Your Linters are passing successful:heavy_check_mark:
   * #### You have a proper Pull Request description that contain a summary of the concepts you learned in completing this project. :heavy_check_mark:
   * #### Nice work to commit oftentimes :heavy_check_mark:
  

### Optional suggestions
  * #### N/a

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
    
Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

#### _**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**_
------------------------------------------------------------------------------------------------------------------------------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

**Pull request merged**